Tasks/UsePythonVersionV0/task.json

```diff
@@ -35,6 +35,14 @@
             "defaultValue": "3.x",
             "helpMarkDown": "Version range or exact version of a Python version to use, using semver's version range syntax. [More information](https://go.microsoft.com/fwlink/?LinkID=2006180)"
         },
+        {
+            "name": "allowUnstable",
+            "type": "boolean",
+            "label": "Allow downloading unstable releases",
+            "required": true,
```

Comment:
Do we really need this as required? Would this be a breaking change then?

Reply:
Yeah, fixed in https://github.com/microsoft/azure-pipelines-tasks/pull/15820/commits/a59284dcbc4c19108b4da367953846d2154bed32. This was a copy-paste error.